components/Cards/src/MDCCard.h

```diff
@@ -113,4 +115,10 @@
  */
 - (nullable UIColor *)shadowColorForState:(UIControlState)state UI_APPEARANCE_SELECTOR;
 
+/*
+ The shape generator used to define the card's shape.
+ */
+@property(nullable, nonatomic, strong) id<MDCShapeGenerating> shapeGenerator UI_APPEARANCE_SELECTOR;
```

Comment:
May be wise for us to avoid increasing our UIAppearance support surface area here - can you remove the `UI_APPEARANCE_SELECTOR`?

Reply:
removed